zone/attack.cpp

```diff
@@ -1042,6 +1042,20 @@ void Mob::MeleeMitigation(Mob *attacker, DamageHitInfo &hit, ExtraAttackOptions
 
 	auto roll = RollD20(hit.offense, mitigation);
 
+	// Add bonus to roll if level difference is sufficient
+	signed lvldiff = attacker->GetLevel() - this->GetLevel();
```

Comment:
`signed`? Why not just use `int`? Also don't need `this`. Can name the variable `level_diff`.